CMakeLists.txt

```diff
@@ -197,6 +197,7 @@ message(STATUS "Written ${PROJECT_BINARY_DIR}/json_config.h")
 
 if ("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU")
     set(CMAKE_C_FLAGS           "${CMAKE_C_FLAGS} -ffunction-sections -fdata-sections")
+    set(CMAKE_C_FLAGS           "${CMAKE_C_FLAGS} -fprofile-arcs -ftest-coverage")
```

Comment:
This is going to add overhead and change the behavior (i.e. result in .gcda files being emitted) and thus can't be enabled in a regular build.

Reply:
You are right, it's not suitable to add "-fprofile-arcs -ftest-coverage" in this regular build. I will try to make it in a new way.